cardano-testnet/src/Testnet/Blockfrost.hs

```diff
@@ -0,0 +1,268 @@
+    bfgMaxTxExSteps         <- o .:# "max_tx_ex_steps"
+    bfgMaxValueSize         <- o .:# "max_val_size"
+    bfgPriceMem             <- o .:# "price_mem"
+    bfgPriceSteps           <- o .:# "price_step"
+
+    -- Conway params
+    bfgCommitteeMaxTermLength     <- o .:# "committee_max_term_length"
+    bfgCommitteeMinSize           <- o .:# "committee_min_size"
+    bfgDRepActivity               <- o .:# "drep_activity"
+    bfgDRepDeposit                <- o .:# "drep_deposit"
+    bfgDVTCommitteeNoConfidence   <- o .:# "dvt_committee_no_confidence"
+    bfgDVTCommitteeNormal         <- o .:# "dvt_committee_normal"
+    bfgDVTHardForkInitiation      <- o .:# "dvt_hard_fork_initiation"
+    bfgDVTMotionNoConfidence      <- o .:# "dvt_motion_no_confidence"
+    bfgDVTPPEconomicGroup         <- o .:# "dvt_p_p_economic_group"
+    bfgDVTPPGovGroup              <- o .:# "dvt_p_p_gov_group"
+    bfgDVTPPNetworkGroup          <- o .:# "dvt_p_p_network_group"
+    bfgDVTPPTechnicalGroup        <- o .:# "dvt_p_p_technical_group"
+    bfgDVTTreasuryWithdrawal      <- o .:# "dvt_treasury_withdrawal"
+    bfgDVTUpdateToConstitution    <- o .:# "dvt_update_to_constitution"
+    bfgGovActionDeposit           <- o .:# "gov_action_deposit"
+    bfgGovActionLifetime          <- o .:# "gov_action_lifetime"
+    bfgMinFeeRevScriptCostPerByte <- o .:# "min_fee_ref_script_cost_per_byte"
+    bfgPVTCommitteeNoConfidence   <- o .:# "pvt_committee_no_confidence"
+    bfgPVTCommitteeNormal         <- o .:# "pvt_committee_normal"
+    bfgPVTHardForkInitiation      <- o .:# "pvt_hard_fork_initiation"
+    bfgPVTMotionNoConfidence      <- o .:# "pvt_motion_no_confidence"
+    bfgPVTPPSecurityGroup         <- o .:# "pvt_p_p_security_group"
+
+    -- Shelley params
+    bfgA0                     <- o .:# "a0"
+    bfgDecentralisationParam  <- o .:# "decentralisation_param"
+    bfgEMax                   <- o .:# "e_max"
+    bfgExtraEntropy           <- o .:# "extra_entropy"
+    bfgKeyDeposit             <- o .:# "key_deposit"
+    bfgMaxBlockHeaderSize     <- o .:# "max_block_header_size"
+    bfgMaxBlockSize           <- o .:# "max_block_size"
+    bfgMaxTxSize              <- o .:# "max_tx_size"
+    bfgMinFeeA                <- o .:# "min_fee_a"
+    bfgMinFeeB                <- o .:# "min_fee_b"
+    bfgMinPoolCost            <- o .:# "min_pool_cost"
+    bfgMinUTxO                <- o .:# "min_utxo"
+    bfgNOpt                   <- o .:# "n_opt"
+    bfgPoolDeposit            <- o .:# "pool_deposit"
+    bfgProtocolMajorVer       <- o .:# "protocol_major_ver"
+    bfgProtocolMinorVer       <- o .:# "protocol_minor_ver"
+    bfgRho                    <- o .:# "rho"
+    bfgTau                    <- o .:# "tau"
+
+    -- Cost models
+    models' <- o .: "cost_models_raw"
+    let models = CostModels.costModelsValid models'
+        bfgAlonzoCostModels = CostModels.mkCostModels $
+          Map.filterWithKey (\lang _ -> lang `elem` [PlutusV1, PlutusV2]) models
```

Comment:
```suggestion
          Map.filterKeys (\lang -> lang `elem` [PlutusV1, PlutusV2]) models
```

Reply:
Perhaps we would need `containers-0.8` for that. You can ignore this if it's too much hassle.

But we definitely should use `Data.Map.Strict`.